Packs/qualys/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Qualys v2
+- Added qualys-update-unix-record command.
+- Updated the Docker image to: *demisto/python3:3.10.4.29342*.
```

Comment:
Please change the docker to:

```suggestion
- Updated the Docker image to: *demisto/python3:3.10.5.31928*.
```

Reply:
Hi, I made the changes in the following commit: https://github.com/demisto/content/pull/20142/commits/9606eb3af1a5d3a4d3c4acaa2b7012eb66d72aa4#diff-5f633019716ea0c1af46dc0c4be364284b7e67bd76f5e0b1a12dd1783fa9c294